composer.json

```diff
@@ -10,16 +10,17 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "illuminate/support": "^5.4"
+        "php": ">=7.2",
+        "illuminate/support": "^6.0"
```

Comment:
You should be able to do `^5.4 | ^6.0`

Reply:
The Lumen framework from version 6.0 requires at least PHP 7.2, that's why I thought this package also need to require PHP 7.2.
I think if somebody don't want to upgrade for a newer PHP version, he will be able to use an older version of lumen-cors.

Reply:
Yes, but then we'd have two separate versions to maintain. I would prefer if you just changed it to work with both new and old versions of `illuminate/support`, that way there's no backward-incompatible changes.

Reply:
Ok, composer.json updated